### ISSUE TITLE
Not working in ui-lovelace-minimalist dashboard 

### ISSUE THREAD

**Issue by faizpuru** [gadgetchnnel/lovelace-header-cards#7]
Thanks for this card ! It's awesome. I was using it flawlessly in my dashboard. I migrate my setup to ui-lovelace-minimalist and your card isn't working anymore.
I've an error in the console : 
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'config')
    at lovelace-header-cards.js?hacstag=466196192009:1:3318

**Comment by warmfire540** [comment]
yeah i'm getting same error.. I had it load once or twice after a bunch of refreshes, but mostly the config error

**Comment by warmfire540** [comment]
ended up making a card that works in UI minimalist.  made custom config so it doesn't require any custom components, just button-card..


My card works slightly different and is not setup globally like this one, but it's very quick and is based on labels/areas instead of config in the card.

https://community.home-assistant.io/t/toolbar-status-chips-card-with-sticky-header-and-driven-by-entities/825075

https://github.com/warmfire540/toolbar-status-chips
